include/cudaq/Optimizer/CodeGen/Passes.h

```diff
@@ -29,7 +29,7 @@ void registerConvertToQIRPass();
 
 /// Convert (generic) QIR to the profile-specific QIR for a specific target.
 /// @param pm Pass Manager to add QIR passes to
-/// @param convertTo Expected to be "qir-base" or "qir-adaptive" (comes from the
+/// @param convertTo Expected to be `qir-base` or `qir-adaptive` (comes from the
```

Comment:
Similar to a nit in another review ;), these terms are not code artifacts and shouldn't be quoted as if they were code artifacts to get around limitations in the spelling checker.

Reply:
> Similar to a nit in another review ;), these terms are not code artifacts and shouldn't be quoted as if they were code artifacts to get around limitations in the spelling checker.

This one feels like a gray area to me. One could argue that since the code accepts these two strings, they are indeed code artifacts. Do you have a counter suggestion (other that removing all spell checking for items in quotes)?